tinker/static/assets/js/tinker.js

```diff
@@ -14,4 +14,130 @@ $(document).ready(function () {
+    if (type === "event") {
+        viewList = ['My Events', 'All Events', 'Other Events', 'User Events'];
+        $("#search-box").empty();
+        $("#search-box").append("<label for=\"event-title\">Event Title:</label>\n" +
+            "<input id=\"event-title\" type=\"text\" name=\"event-title\"class=\"input-field\" value=\"" + inputs.split("-")[0] + "\"><br/>\n" +
+            "<label for=\"start-date\">Date Range:<br />\n" +
+            "<span>\n" +
+            "<input id=\"start-date\" class=\"datepicker input-field date-range\" size=\"11\" value=\"" + inputs.split("-")[1] +" \">\n" +
+            "<input id=\"end-date\" class=\"datepicker input-field date-range\" size=\"11\" value=\"" + inputs.split("-")[2] + " \">\n" +
+            "</span>\n" +
+            "</label>")
+    } else if (type === "e-announcement") {
+        viewList = ['My E-Announcements', 'All E-Announcements', 'Other E-Announcements', 'User E-Announcements'];
+        $("#search-box").empty();
+        $("#search-box").append("<label for=\"e-annz-title\">E-Announcement Title:</label>\n" +
+            "<input id=\"e-annz-title\" type=\"text\" name=\"E-Annz-Title\" class=\"input-field\" value=\"" + inputs.split("-")[0] + "\">\n" +
+            "<label for=\"e-annz-date\">E-Announcement Date</label>\n" +
+            "<input id=\"e-annz-date\" class='datepicker input-field date-range' size='11' value=\"" + inputs.split("-")[1] + "\">")
+    }
+
+    let lastInput = inputs.split("-").length - 1;
+    let selected = 0;
+    if (!inputs.split("-")[lastInput]) {
+        selected = $("#school-selector").children("option:selected").attr("value");
+    }
+    $("#school-selector").empty();
+    if ((type === "event" && (groups.includes('Tinker Events - CAS') || groups.includes('Event Approver'))) || (type === "e-announcement" && (groups.includes('Tinker E-Announcements - CAS') || groups.includes('E-Announcement Approver')))) {
+        for (let i = 0; i < viewList.length - 1; i ++) {
+            if (inputs.split("-")[lastInput] - 1 == i || selected - 1 == i) {
+                $("#school-selector").append("<option value=\""+ (i + 1) +"\" selected='selected'>" + viewList[i] + "</option>");
+            } else {
+                $("#school-selector").append("<option value=\""+ (i + 1) +"\">" + viewList[i] + "</option>");
+            }
+        }
+    } else {
+        if (type === "event") {
+            $("#school-selector").append("<option value=\""+ 1 +"\" selected='selected'>" + viewList[lastInput] + "</option>");
+        } else {
+            $("#school-selector").append("<option value=\""+ 1 +"\" selected='selected'>" + viewList[lastInput + 1] + "</option>");
+        }
+    }
+
+
+    $('.datepicker').each(function () {
+        var current_element = $(this);
+        var starting_values = {
+            field: this,
+            format: 'MM DD YYYY',
+            minDate: new Date('2000-01-01'),
+            maxDate: new Date('2040-12-31'),
```

Comment:
minDate and maxDate should be specific dates. minDate can be something like 2010. MaxDate should be 2 years from today.

Reply:
this was existing code that I just copied and pasted here. Do you want me to also change the values in the other places that it occurs?

Reply:
Yeah I think so.

Reply:
Fixed